docs/resources/connector.md

```diff
@@ -24,6 +24,7 @@ resource "twingate_remote_network" "aws_network" {
 
```

Comment:
Can you update the doc to show what the default value is for `status_updates_enabled`

Reply:
I made the change